cluster-autoscaler/core/scaledown/planner/planner.go

```diff
@@ -289,14 +292,15 @@ func (p *Planner) categorizeNodes(podDestinations map[string]bool, scaleDownCand
 		// TODO(x13n): break on timeout. Figure out how to handle nodes
 		// identified as unneeded in previous iteration, but now
 		// skipped due to timeout.
-		removable, unremovable := p.rs.SimulateNodeRemoval(node, podDestinations, p.latestUpdate, pdbs)
+		removable, unremovable := p.rs.SimulateNodeRemoval(node, podDestinations, p.latestUpdate, p.pdbRemainingDisruptions.GetPdbs())
 		if unremovable != nil {
 			unremovableCount += 1
 			p.unremovableNodes.AddTimeout(unremovable, unremovableTimeout)
 		}
 		if removable != nil {
 			delete(podDestinations, removable.Node.Name)
 			removableList = append(removableList, *removable)
+			p.pdbRemainingDisruptions.Update(removable.PodsToReschedule)
```

Comment:
Please don't ignore the error returned by `Update`. This should be logged at error severity.